tea/include/extension.h

```diff
@@ -35,7 +35,11 @@ typedef zend_result zend_result_t;
 #endif
 
 typedef zend_result_t (*tea_extension_init_function)(INIT_FUNC_ARGS);
-typedef zend_result_t (*tea_extension_shutdown_function)(SHUTDOWN_FUNC_ARGS); /* }}} */
+typedef zend_result_t (*tea_extension_shutdown_function)(SHUTDOWN_FUNC_ARGS);
+typedef void (*tea_extension_op_array_function)(zend_op_array *op_array);
+typedef zend_result_t (*tea_extension_startup_function)(void); /* }}} */
+
+zend_module_entry* tea_extension_module();
```

Comment:
This should not be leaked.

Reply:
How would you prefer to write the test then?

https://github.com/DataDog/dd-trace-php/pull/1503/files#diff-5f9e4cde6523629a254e406c13d930a7bb1a29131ad64044e573a6212fe0495aR41